.github/workflows/integrationtests.yml

```diff
@@ -10,6 +10,10 @@ jobs:
   test:
     runs-on: ubuntu-latest
 
+    env:
```

Comment:
This change is not required

    env:
      CHAT_SERVICE_PORT: 8000
      OLLAMA_PORT: 11434


Remove it